tests/batch/autoscaler_test.py

```diff
@@ -74,6 +74,7 @@ def test_run_ok(mock_sensu, mock_running, mock_time, mock_sleep, dry_run):
     with mock.patch('builtins.hash') as mock_hash:
         mock_hash.return_value = 0  # patch hash to ignore splaying
         batch_obj.run()
+
```

Comment:
I don't really care whether this space is there or not, but usually if whitespace is the only change in the file I will amend the commit to make the diff smaller.  Just FYI, don't have to block on this or whatever.